src/Home.js

```diff
@@ -1,17 +1,111 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, Text, View, FlatList, Pressable } from 'react-native';
+import { DataStore } from 'aws-amplify';
+import { Todo } from './models';
 
 export default function Home() {
+
+    const [todos, setTodos] = useState([]);
+
+    let subscription;
+
+    useEffect(() => {
+        onQuery = async () => {
+            const todos_list = await DataStore.query(Todo);
+            setTodos(todos_list);
+        };
+        onQuery();
```

Comment:
Couple questions:
- where is `onQuery` initialized?
- why do we need to call it?

Reply:
`onQuery` is initialized inside the `useEffect()`. There are two places it is being called. First is inside the `useEffect()` where it will load the todoList for the first render. Second, it is being called inside the function passed to the subscription. This function will be called every-time a new subscription is received from the cloud which indicates change in the todo list.

Reply:
Maybe I am missing something here, but I do not see `let onQuery;` in this file. That being said it does not appear that we need to reassign it, so maybe just declaring it as a `const` here would be sufficient?

Regarding the immediate call of `onQuery` after its declaration, maybe a comment along the lines of "load todos on first render" or something similar would help?

Reply:
I updated the const onQuery. That was a good catch. I have added the comment as well.